`lxc alias add` panics if run on uninitialised host

```
root@vtest:~# snap install lxd --channel=5.0/candidate
root@vtest:~# lxc alias add delete "delete -i"
If this is your first time running LXD on this machine, you should also run: lxd init
To start your first container, try: lxc launch ubuntu:22.04
Or for a virtual machine: lxc launch ubuntu:22.04 --vm

panic: assignment to entry in nil map

goroutine 1 [running]:
main.(*cmdAliasAdd).Run(0xc0000fd370, 0x0?, {0xc0001b4d20?, 0x2, 0x0?})
	/build/lxd/parts/lxd/src/lxc/alias.go:84 +0xdd
github.com/spf13/cobra.(*Command).execute(0xc000164500, {0xc0001b4cc0, 0x2, 0x2})
	/build/lxd/parts/lxd/src/.go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:872 +0x694
github.com/spf13/cobra.(*Command).ExecuteC(0xc000164000)
	/build/lxd/parts/lxd/src/.go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:990 +0x3b4
github.com/spf13/cobra.(*Command).Execute(...)
	/build/lxd/parts/lxd/src/.go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:918
main.main()
	/build/lxd/parts/lxd/src/lxc/main.go:276 +0x1a33
```

Reported from https://discuss.linuxcontainers.org/t/panic-with-lxc-alias/14911/2

--- a/lxc/config/config.go
+++ b/lxc/config/config.go
@@ -81,11 +81,14 @@ func (c *Config) SaveCookies() {
 
 // NewConfig returns a Config, optionally using default remotes.
 func NewConfig(configDir string, defaults bool) *Config {
-	config := &Config{ConfigDir: configDir}
+	var config *Config
 	if defaults {
-		config.Remotes = DefaultConfig.Remotes
-		config.DefaultRemote = DefaultConfig.DefaultRemote
+		config = DefaultConfig()
+	} else {
+		config = &Config{}
 	}
+
+	config.ConfigDir = configDir
 
 	return config
 }

--- a/lxc/config/default.go
+++ b/lxc/config/default.go
@@ -45,8 +45,17 @@ var DefaultRemotes = map[string]Remote{
 	"ubuntu-daily": UbuntuDailyRemote,
 }
 
-// DefaultConfig is the default configuration.
-var DefaultConfig = Config{
-	Remotes:       DefaultRemotes,
-	DefaultRemote: "local",
+// DefaultConfig returns the default configuration.
+func DefaultConfig() *Config {
+	// Duplicate remotes from DefaultRemotes.
+	defaultRoutes := make(map[string]Remote, len(DefaultRemotes))
+	for k, v := range DefaultRemotes {
+		defaultRoutes[k] = v
+	}
+
+	return &Config{
+		Remotes:       defaultRoutes,
+		Aliases:       make(map[string]string),
+		DefaultRemote: "local",
+	}
 }

--- a/lxc/config/file.go
+++ b/lxc/config/file.go
@@ -72,7 +72,7 @@ func LoadConfig(path string) (*Config, error) {
 	if len(envDefaultRemote) > 0 {
 		c.DefaultRemote = envDefaultRemote
 	} else if c.DefaultRemote == "" {
-		c.DefaultRemote = DefaultConfig.DefaultRemote
+		c.DefaultRemote = DefaultConfig().DefaultRemote
 	}
 
 	// NOTE: Remove this once we only see a small fraction of non-simplestreams users
@@ -105,9 +105,11 @@ func (c *Config) SaveConfig(path string) error {
 		}
 	}
 
+	defaultRemote := DefaultConfig().DefaultRemote
+
 	// Remove the static remotes
 	for k := range StaticRemotes {
-		if k == DefaultConfig.DefaultRemote {
+		if k == defaultRemote {
 			continue
 		}
 

